hw/ip/aes/aes.core

```diff
@@ -24,8 +24,58 @@ filesets:
       - rtl/aes.sv
     file_type: systemVerilogSource
 
+  files_verilator_waiver:
+    depend:
+      # common waivers
+      - lowrisc:lint:common
+      - lowrisc:lint:comportable
+    files:
+      - lint/aes.vlt
+    file_type: vlt
+
+  files_ascentlint_waiver:
+    depend:
+      # common waivers
+      - lowrisc:lint:common
+      - lowrisc:lint:comportable
+      # tool setup
+      - lowrisc:lint:ascentlint
```

Comment:
You can include that in lint:common to avoid adding that line in all cores.

Reply:
Makes sense, yes.

Reply:
done